CPS.md

```diff
@@ -559,7 +555,7 @@ Investigation into a revocation request will begin within 24 hours of receiving
 
 ### 4.9.6 Revocation checking requirement for relying parties
 
-Relying Parties who cannot or choose not to check revocation status, but decide to rely on a certificate anyway, do so at their own risk.
+Relying parties should verify the validity of certificates via CRL or OCSP prior to relying on certificates. Relying Parties who cannot or choose not to check certificate expiration or revocation status, but decide to rely on a certificate anyway, do so at their own risk.
```

Comment:
Still need to fix capitalization of "parties" in the second word of this paragraph.

Reply:
Fixed